fetch-schema.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+require('dotenv').config({ path: '.env.local' })
+const exec = require('child_process').exec
+const dataConfig = require('./relay.config')
+const thegraphConfig = require('./relay_thegraph.config')
+/* eslint-enable */
+
+exec(`get-graphql-schema https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3 > ${thegraphConfig.schema}`)
+
+const API_URL = process.env.REACT_APP_GQL_API_URL
+const API_KEY = process.env.REACT_APP_GQL_API_KEY
+console.log(process.env)
+
+if (API_URL && API_KEY) {
+  exec(
+    `get-graphql-schema https://dsn76qqamreobir5plkvauf4dm.appsync-api.us-east-2.amazonaws.com/graphql --h X-API-KEY=da2-n65b7e42gzgxzbye4f366ukss4 > ${dataConfig.schema}`
```

Comment:
is this the api key you don't want in source control?

Reply:
OOOPS will do

Reply:
I was using it dynamically but hardcoded it while debugging whoops